website/contents/pages.js

```diff
@@ -481,7 +481,7 @@ export const docPages = generatePath([
       },
       {
         name: 'TestRenderer',
-        content: getDocUrl('api-reference/test-utils/test-renderer.md')
+        content: getDocUrl('api-reference/test-utils/renderer-test.md')
```

Comment:
If the class is named TestRenderer let's name the file `test-renderer.md`

Reply:
The class is named as RenderTest

Reply:
Then you should update the name of this page...